examples/webgl-overlay/__tests__/index.js

```diff
@@ -0,0 +1,25 @@
+const { join } = require('path');
+
+it('should have consistent performance', async () => {
```

Comment:
I'm not sure this is doing what you expect it to be doing.

If it was just copy/pasted, I'm happy to see it deleted. If it was intentional, a few things will need fixing up -

* We need the code which reads the rendered canvas pixel data back as it is the only way to accurately measure the performance. 
* We also need to make sure the clicking that's been done is meaningful to the test.